### ISSUE TITLE
Update checker errors and fails

### ISSUE THREAD

**Issue by AmericanCraft** [Pinelog-Studios/AdditionalGuns#27]
I found another console issue where the update checker errors and fails.
A friend of mine told me this is because of a corrupt file on the mods end.

```
[25Mar2022 12:22:02.185] [Forge Version Check/INFO] [net.minecraftforge.fml.VersionChecker/]: [additionalguns] Starting version check at https://raw.githubusercontent.com/Mr-Pineapple/ModUpdateJSON/master/additionalguns/update.json
[25Mar2022 12:22:02.892] [Forge Version Check/WARN] [net.minecraftforge.fml.VersionChecker/]: Failed to process update information
com.google.gson.JsonSyntaxException: com.google.gson.stream.MalformedJsonException: Expected name at line 5 column 4 path $..0.1.0
	at com.google.gson.Gson.fromJson(Gson.java:947) ~[gson-2.8.8.jar%2322!/:?]
	at com.google.gson.Gson.fromJson(Gson.java:897) ~[gson-2.8.8.jar%2322!/:?]
	at com.google.gson.Gson.fromJson(Gson.java:846) ~[gson-2.8.8.jar%2322!/:?]
	at com.google.gson.Gson.fromJson(Gson.java:8[17](https://aternos.org/log/#L17)) ~[gson-2.8.8.jar%2322!/:?]
	at net.minecraftforge.fml.VersionChecker$1.process(VersionChecker.java:[18](https://aternos.org/log/#L18)9) ~[fmlcore-1.18.1-39.1.2.jar%2343!/:?]
	at java.lang.Iterable.forEach(Iterable.java:75) [?:?]
	at net.minecraftforge.fml.VersionChecker$1.run(VersionChecker.java:128) [fmlcore-1.18.1-39.1.2.jar%2343!/:?]
Caused by: com.google.gson.stream.MalformedJsonException: Expected name at line 5 column 4 path $..0.1.0
	at com.google.gson.stream.JsonReader.syntaxError(JsonReader.java:1562) ~[gson-2.8.8.jar%23[22](https://aternos.org/log/#L22)!/:?]
	at com.google.gson.stream.JsonReader.doPeek(JsonReader.java:504) ~[gson-2.8.8.jar%[23](https://aternos.org/log/#L23)22!/:?]
	at com.google.gson.stream.JsonReader.hasNext(JsonReader.java:413) ~[gson-2.8.8.jar%2[32](https://aternos.org/log/#L32)2!/:?]
	at com.google.gson.internal.bind.ObjectTypeAdapter.read(ObjectTypeAdapter.java:69) ~[gson-2.8.8.jar%2322!/:?]
	at com.google.gson.internal.bind.TypeAdapterRuntimeTypeWrapper.read(TypeAdapterRuntimeTypeWrapper.java:41) ~[gson-2.8.8.jar%2322!/:?]
	at com.google.gson.internal.bind.MapTypeAdapterFactory$Adapter.read(MapTypeAdapterFactory.java:187) ~[gson-2.8.8.jar%2322!/:?]
	at com.google.gson.internal.bind.MapTypeAdapterFactory$Adapter.read(MapTypeAdapterFactory.java:145) ~[gson-2.8.8.jar%2322!/:?]
	at com.google.gson.Gson.fromJson(Gson.java:932) ~[gson-2.8.8.jar%2322!/:?]
	... 6 more
```

(This happens on both client and server)

**Comment by Autovw** [comment]
I've created a [PR](https://github.com/Mr-Pineapple/ModUpdateJSON/pull/1) to fix this a couple months ago, best solution would be to move the update checker to a place both @Mr-Pineapple and I have access to.

**Comment by Autovw** [comment]
This issue has finally been fixed in 0.7.2 :)
